app/views/content/start-teacher-training-this-september.md

```diff
@@ -30,8 +30,8 @@ Our teacher training advisers can help you write a strong application quickly.
 They will help you to:
```

Comment:
Check line 28 for potential duplicated content - this relates to my Q in the initial PR comments.